mantidproject/mantid
# Documentation: math mode use in Mantid docs

This PR is a result of investigation of Latex and math-mode use in Mantid documentation started by #30679 . The use of roman vs italic characters in math-mode of units and functions has been corrected wherever spotted. Intended use of math in algorithm property descriptions using <math></math> blocks has been replaced with form readable by Sphinx. Spotted issues in references and links have also been corrected.

The one spotted issue that this PR does not address is the observed issue with default parameters for certain dbl list properties that are not broken during creation of restructured text by Sphinx and result in overflowing horizontal dimensions of the document. One of the examples is  [EnggFitPeaks](https://docs.mantidproject.org/nightly/algorithms/EnggFitPeaks-v1.html) algorithm.

List of changes by category is as follows:
1. Corrected algorithm property description:

`GetEi2`, `ConvertToDiffractionMDWorkspace`, `ConvertToMD` 

2. Changed units and functions to roman:

`AnvredCorrection`, `EnggCalibrateFull`, `He3TubeEfficiency`, `IntegrateEllipsoids`, `IntegratePeaksMD`-v2, `LorentzCorrection`, `SpecularReflectionPositionCorrect`, `DiffRotDiscreteCircle`, `EISFDiffCylinder`, `GramCharlier`, `HallRoss`, `InelasticDiffRotDiscreteCircle`, `Meier`, `MultivariateGaussianComptonProfile`, `Voigt`

3. Corrected or improved references to either bibliography or other algorithms:

`D7YIGPositionCalibration`, `DetectorEfficiencyCorUser`, `DivideMD`, `LoadVesuvio`, `NormaliseByDetector`, `RefinePowderInstrumentParameters`, `PolarisedDiffractionILL`

4. Other formatting changes:

`ComputeIncoherentDOS`, `ConvertCWSDExpToMomentum`, `GetEi`-v2, `XrayAbsorptionCorrection`, `FickDiffusion`, `LogNormal`, `GenerateEventsFilter`

**To test:**

1. Build the documentation with docs-html as target
2. Make sure the introduced changes are well formatted and work as described.



Fixes #30704. 

*This does not require release notes* because it is a change of formatting of math mode and corrections of typos in documentation.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
